spec/models/sample_spec.rb

```diff
@@ -167,4 +167,79 @@
       expect(sample.sample_metadata.supplier_name).to be_nil
     end
   end
+
```

Comment:
Can I request that we remove all mention of 'parent' and 'child' from this? I think it's inherently confusing. 
- I would say that it's intuitive for the multiple samples to be referred to as the 'parents' (because they existed first), and the single, compound sample to be the 'child' (because they arose out of the multiple samples).
- However, you are using the terms the opposite way round here.

Therefore I suggest that we refer to them consistently as 'compound sample' and 'component samples'.

Reply:
Nice set of test cases 👍